scripts/parts.py

```diff
@@ -30,8 +30,9 @@
                         dest='mapr_cols', default='all')
     parser.add_argument('--levels-testing', help="Set to test filename if correlator levels not accessible [levels.tst]", 
                         dest='levels_testing', default='levels.tst')
-    parser.add_argument('--date', help="MM/DD/YY or now [now]", default='now')
-    parser.add_argument('--time', help="hh:mm or now [now]", default='now')
+    parser.add_argument('--date', help="UTC YYYY/M/D or now [now]", default='now')
+    parser.add_argument('--time', help="UTC hh:mm or now [now]", default='now')
```

Comment:
If many scripts are going to have these kinds of similar/identical arguments, it might be helpful to add a utility module that has functions like `add_datetime_args(parser)` to reduce the code duplication.